src/v3/test/integration/authenticator-enroll-data-phone.test.tsx

```diff
@@ -79,21 +79,6 @@ describe('authenticator-enroll-data-phone', () => {
     );
   });
 
-  it('phone number and extension fields should be ltr even when a rtl language is set', async () => {
```

Comment:
No need for this one any more since it doesn't matter whether it's LTR or RTL with the stacked layout.